src/Payments/Integrations/CashOnDelivery.php

```diff
@@ -70,7 +70,11 @@ private function get_enable_for_virtual() {
 	 * @return array Array of shipping methods (string ids) that allow COD. (If empty, all support COD.)
 	 */
 	private function get_enable_for_methods() {
-		return $this->get_setting( 'enable_for_methods', [] );
+		$enable_for_methods = $this->get_setting( 'enable_for_methods', [] );
+		if ( '' === $enable_for_methods ) {
```

Comment:
Nice - I wonder if there is a standard way to enforce array return value? 

For example, we might want to coerce/cast or call a validate function to ensure that the option value is ALWAYS converted to `array`. Similar to `filter_var( ..., FILTER_VALIDATE_BOOLEAN )`, but for array type (note - `filter_var` doesn't support this!).

The empty string check works fine for this case, I'm just wondering if we can make it more robust.